paper_planes.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>sanjoseallday</title>
+  </head>
+  <body>
+    <h1>You have arrived at Paper Planes</h1>
+    <p>
+      Walking up to the bar you see a long line to enter the bar. The security
+      guards are checking patrons ID's at the door. Everyone waiting looks very
+      excited to get in and have a drink. As you step over the tram track
+      towards the back of the line you hear a horn whistle from the tram
+      approaching. You step aside the tracks to allow the tram passage.
+    </p>
+    <p>
+      Everything around you seems very lively. You notice the crowd looks to be
+      a lot of younger people with trending fashion sense. Some people in line
+      are talking about their college classes, and some are talking about how
+      incredible the drink was that they had last weekend at Paper Planes. As
+      the line shortens and you near the door you begin to smell something
+      amazing. A few minutes ago while you were waiting and socializing in line,
+      a few hot dog stands have set up all along the block offering drunk bar
+      goers a delicious reprieve from hunger.
+    </p>
+    <p>
+      The line was long but you find that time just flies by as you converse
+      with the fellow bar goers waiting with you and now youre getting your I-D
+      checked by a stern yet proffessional security gaurd. after close thorough
```

Comment:
Professional is misspelled, after should be capitalized, and "after close thorough?".